its/projects/NuGet.config

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+      <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
+  </packageSources>
+  <trustedSigners>
+    <repository name="nuget.org" serviceIndex="https://api.nuget.org/v3/index.json">
```

Comment:
I'm not very happy with having these certificates and configurations all over the place for maintenance reasons. At the same time, moving `analyzers/NuGet.config` to the root of the repo would be ideal solution, but too difficult for purpose of this PR.

What about using same trick as there's in .NET ITs here too?
https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/sources/NuGet.config

It's not as nice as having all dependencies listed. At the same time, we already use it in .NET ITs, and Java ITs should not be different (in one way, or the other)

Reply:
I think it's better to keep an eye on the installed packages. While investigating the errors I also found a reference to https://www.nuget.org/packages/Microsoft.VisualStudio.UnitTesting that maybe we should reevaluate.

Reply:
I would suggest to move towards whitelisting all dependencies, for all ITs. 

Java ITs are simpler, so it's easier to do it now.
We will need to do that for .NET ITs, too.

ITs builds run on our build infra, we should treat them seriously for their supply chain risk.

Reply:
and I'd prefer to have separate `nuget.config` with trusted signers between ITs and build, to have a clear picture. 

One solution could be having all integration tests together - then we'd have only two nuget.configs - one for ITs, one for the build.

There are multiple maintainability issues between our two sets of ITs, bigger than nuget configs (which, in practice, we rarely touch).

It's a good topic to discuss, maybe outside of this PR and looking at the big picture.

Reply:
Everything that we restore is cached and will be reused in the following runs. I think it's safer to have strict rules.

Reply:
In the spritit of

>(in one way, or the other)

we need to update the .NET part then.

Reply:
- What do you mean by .NET part? I don't want to have a reunion. I only want to enumerate the required owners by the ITs
- It's not in the scope of this PR. The goal of this PR is to fix the errors we have on CI.